.gitattributes

```
@@ -0,0 +1 @@
+*.csv eol=lf
```

Do we need to add a line here for `wait-for-it.sh` as well?
```suggestion
*.csv eol=lf
wait-for-it.sh eol=lf
```